### ISSUE TITLE
build jit with --prod

### ISSUE THREAD

**Issue by BubbleLeaf** [angular/angular-cli#4904]
### OS
> Windows 7, 8 or 10

### Versions.
@angular/cli: 1.0.0-beta.31
node: 6.9.5
os: win32 x64
@angular/common: 2.4.8
@angular/compiler: 2.4.8
@angular/core: 2.4.8
@angular/forms: 2.4.8
@angular/http: 2.4.8
@angular/platform-browser: 2.4.8
@angular/platform-browser-dynamic: 2.4.8
@angular/router: 3.4.8
@angular/flex-layout: 2.0.0-beta.5
@angular/cli: 1.0.0-beta.31
@angular/compiler-cli: 2.4.8

### Message
I found ```--aot defaults to true in --prod``` at beta.28,so how can I build my web in --prod with JIT . I need compiler to do something in the runtime . there is no '--jit' flag I found to use.

**Comment by clydin** [comment]
try `--no-aot`.  Allow boolean options can be negated using the `--no-` form.

**Comment by BubbleLeaf** [comment]
it works , thanks for your help .

**Comment by sebelga** [comment]
Hi, I am reopening the issue because in my case it keeps complaining about lambda function.
I use:

```
ng build --no-aot --prod
````

And I receive this error:
ERROR in Error encountered resolving symbol values statically. Function calls are not supported. Consider replacing the function or lambda with a reference to an exported function

Thanks for the help!

**Comment by captainkovalsky** [comment]
@sebelga I tried to run with --no-aot and it helps me at first run. When I run the same command again I got the error about `encountered resolving symbol values statically`. I've investigated the @angular/cli documentation carefully and couldn't find any references about --no flag. Just found this one https://github.com/angular/angular-cli/commit/e15433e74a8832f5eec57df5776894a19dd5d344#diff-38fa56d60acdab051b2b22316bdf16acR62
